products/podman.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
+versionCommand: podman --remote version --format '{{.Server.Version}}'
+changelogTemplate: "https://github.com/containers/podman/releases/tag/v__LATEST__"
+releaseDateColumn: true
+
+identifiers:
+-   repology: podman
+
+auto:
+  methods:
+  -   git: https://github.com/containers/podman.git
+
+releases:
+-   releaseCycle: "5.2"
+    releaseDate: 2024-08-02
+    eol: false
+    latest: "5.2.0"
+    latestReleaseDate: 2024-08-02
+
+-   releaseCycle: "5.1"
+    releaseDate: 2024-05-29
+    eol: true
```

Comment:
Have dates is usually more useful for users. If only the latest release is supported the following rule can be applied: `eol(x) = releaseDate(x+1)`. WDYT ?

Reply:
Some releases are overlapping, so that probably needs some kind of statement from upstream